### PR TITLE
renew lib-sc-prod tls certs

### PR DESCRIPTION
**Pull request by kayiwa**

### CODE REVIEW

File: roles/nginxplus/files/ssl/lib-sc-prod_princeton_edu_chained.pem [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
+V51obps+LTZWJoQU1YvOAHYA6D7Q2j71BjUy51covIlryQPTy9ERa+zraeF3fW0G
+vW4AAAGAim+KsgAABAMARzBFAiEA6/d4vklbhtb5mS8gsLtUfNX5CCD+b2zkNqUt
+/G8VRlwCIHQGoP5mU5qmfPSctDC9SK6HXu8an2kNshm/McpQ0MpwMA0GCSqGSIb3
+DQEBCwUAA4IBAQA1ylHQlaRjlFm0zpoJlh28krJ4HaGyKcPrbUa28YW30zdGeE9P
+ZFY5+KGadPSOg6Hw9EQfQ3Hgp33fl5JmXH09RoezNo/HpHmwgDSvI8sAjFuIo7JH
+lDsfiVd5BPuZou/t4L8eT/GshLyr85vc1DGwQzs7b8eiBc4BexP5VvBtaKKvt51P
+2L4Ot2ImHBqtg28POvNJGn+szloChXg9f6KhIDYoRgz3buVTx4dI+FS1lAebJn5T
+JXrY0/xkXb8Fza8Y1q7LNltGXBAH1aTuKN1+S0IMHIM8ru+qMlTeszuLWZpk4qIA
+6asquhGDNCjqIkcj/h0P5fldfJwiTscZOpFk
 -----END CERTIFICATE-----
```
Comment (acozine) [inline roles/nginxplus/files/ssl/lib-sc-prod_princeton_edu_chained.pem:39]: Shouldn't there be changes to this part of the chained cert too?

**Review by acozine — COMMENTED** [review]
Looked at this with @christinach @carolyncole @sandbergja in the Ansible meeting today and we had questions . . .

File: roles/nginxplus/files/ssl/lib-sc-prod_princeton_edu_chained.pem (hunk 1 above)
Comment (kayiwa) [reply roles/nginxplus/files/ssl/lib-sc-prod_princeton_edu_chained.pem:39]:
```
❯ openssl x509 -in lib-sc-prod_princeton_edu_chained.pem -text
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            83:d6:b5:d6:92:a4:34:1a:5c:57:54:f3:36:b4:d0:a5
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: C=US, ST=MI, L=Ann Arbor, O=Internet2, OU=InCommon, CN=InCommon RSA Server CA
        Validity
            Not Before: May  3 00:00:00 2022 GMT
            Not After : Jun  3 23:59:59 2023 GMT
        Subject: C=US, ST=New Jersey, O=The Trustees of Princeton University, OU=OIT, CN=lib-sc-prod.princeton.edu
```

Comment (kayiwa) [reply roles/nginxplus/files/ssl/lib-sc-prod_princeton_edu_chained.pem:39]: I don't know how to answer that... but it checks out.

Comment (acozine) [reply roles/nginxplus/files/ssl/lib-sc-prod_princeton_edu_chained.pem:39]: Working code wins.

**Pull request merged**